code/game/gamemodes/umbrage/umbrage_abilities/silver_tongue.dm

```diff
@@ -0,0 +1,54 @@
+//Can be used on a communications console to recall the shuttle. Leaves visible evidence.
+/datum/action/innate/umbrage/silver_tongue
+	name = "Silver Tongue"
+	id = "silver_tongue"
+	desc = "When used near a communications console, allows you to forcefully transmit a message to Central Command, initiating a shuttle recall."
+	button_icon_state = "umbrage_silver_tongue"
+	check_flags = AB_CHECK_CONSCIOUS|AB_CHECK_STUNNED
+	psi_cost = 100
+	lucidity_cost = 1 //Very niche, so low cost
+	blacklisted = 0
+
+/datum/action/innate/umbrage/silver_tongue/IsAvailable()
+	if(SSshuttle.emergency.mode != SHUTTLE_CALL)
```

Comment:
As mentioned above, umbrage ones at least need to check for psi so that people don't try to cast them and then screech when they can't. It's a necessary evil, unless you can think of a better way, and why I'm keeping it but just under umbrage abilities.